lib/spack/spack/cmd/uninstall.py

```diff
@@ -21,12 +21,16 @@
 from llnl.util.tty.colify import colify
 
 description = "remove installed packages"
+description += ("\n\nSpecs to be uninstalled are specified using the spec "
+                "syntax (`spack help --spec`) and can be identified by their "
+                "hashes. To remove packages that are needed only at build "
+                "time and were not explicitly installed see `spack gc -h`.")
```

Comment:
Won't this "break" `spack help` in the sense that it will have a super long entry for a single command? For other commands, we use `subparser.epilog` for this reason.

Reply:
@adamjstewart You're right. Moved that and part of the `--all` help message to the epilog.